### DISCUSSION TITLE
False login screen

### DISCUSSION THREAD

**Discussion by DanieruOtakuBoy** [pixelfed/pixelfed-rn#80] [Bug Reports]
Sometimes the(Android) app shows me the login screen when my account is already in the app, it works again after some 'closing and re opening' the app.

![Screenshot_2024-07-05-17-49-55-455_com pixelfed android](https://github.com/pixelfed/pixelfed-rn/assets/130202303/068ae243-27c6-428d-9532-013ac8a1d75c)

**Comment by dansup** [comment]
Fixed in v1.0.0.65
